### DISCUSSION TITLE
I want to give a hand on this project

### DISCUSSION THREAD

**Discussion by ghyatzo** [Gnimuc/CImGui.jl#70] [General]
While working on a project that uses this pacakge I had the chance to get accustomed a bit with the library and some of its structure.
Albeit the whole automatically generated through `CLang.jl` is still abit fuzzy.
I really like where this is going and i think that the julia ecosystem really needs something like this. So I would like to give a hand where possible. Is there something in particular that needs to be done? Also with regards to #52 and https://github.com/Gnimuc/CImGui.jl/discussions/52#discussioncomment-3666538?

**Comment by Gnimuc** [comment]
Feel free to open new PRs if you'd like to add new features. Although I'm not actively developing this package at the moment, I'll try to review and merge new PRs as soon as possible.

**Reply by ghyatzo** [reply]
I'll try to think of something!
